.github/workflows/ci.yml

```diff
@@ -35,6 +35,7 @@ jobs:
       - run: ajv validate -s app/schema.json -d "{docs/examples/viewconfs,test/{view-configs,view-configs-more}}/*.json"
 
   Test:
+    needs: [Lint, Schema]
```

Comment:
These actions take almost no time, and it will prevent the long test suite from running if either fail.